src/diracx/db/sql/utils.py

```diff
@@ -107,7 +109,7 @@ def available_urls(cls) -> dict[str, str]:
                     else:
                         db_urls[db_name] = parse_obj_as(SqlalchemyDsn, db_url)
                 except Exception:
-                    logger.error("Error loading URL %s for %s", db_name, db_url)
+                    logger.error("Error loading URL for %s", db_name)
```

Comment:
```suggestion
                    logger.error(f"Error loading URL for {db_name}")
```

Reply:
This is intentional, see https://github.com/DIRACGrid/diracx/pull/35#discussion_r1251035371